### DISCUSSION TITLE
How get old value when lookup changed

### DISCUSSION THREAD

**Discussion by eissa1005** [serenity-is/Serenity#5772] [Help] [answered]
I want save old value and new value when lookup changed

**Comment by VictorTomaili** [comment]
On the repository, you can access the old row with `Old` property 👴

**Reply by eissa1005** [reply]
I want it here on  this CurrencyID Lookup
 this.form.CurrencyID.changeSelect2(() => {
              
                var edit = Serenity.EditorUtils.getValue(this.form.CurrencyID);
                var CurrencyID = this.form.CurrencyID.value;
                if (Q.isEmptyOrNull(CurrencyID)) {
                    var Currency = Q.tryFirst(CurrencyRow.getLookup().items, x => x.CurrencyID == CurrencyID);
                    this.form.RATE.value = Currency.CUR_RAT;
                }
            });

**Reply by eissa1005** [reply]
I do not mean to save in the data, I mean to store the value before and after the change in a variable

**Comment by eissa1005** [comment, marked as answer]
Resolve**  By Code
   this.form.CurrencyID.changeSelect2((e) => {

                var oldValue = e["removed"].id;
                var newValue = e["added"].id;

            });
